DSCPullServerAdmin/src/CmdLets/RemoveDSCPullServerAdminRegistration.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Isam.Esent.Interop;
+using DSCPullServerAdmin.src.Models;
+using DSCPullServerAdmin.src.Helpers;
+
+namespace DSCPullServerAdmin.src.CmdLets
+{
+    [Cmdlet(VerbsCommon.Remove,
+        "DSCPullServerAdminRegistration")]
+    [OutputType(typeof(void))]
+    public class RemoveDSCPullServerAdminRegistration : BaseCmdlet
+    {
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true)]
+        [Alias("Id")]
+        [ValidateNotNullOrEmpty()]
+        public string AgentId;
```

Comment:
Could you also add a InputObject parameter like with Remove-DSCPullServerAdminReport?
eg:
```
[Parameter(ValueFromPipeline = true,
    ParameterSetName = "InputObject")]
        public RegistrationData InputObject;
```